src/Dropbox.elm

```diff
@@ -817,7 +880,7 @@ decodeWriteConflictError =
 See <https://www.dropbox.com/developers/documentation/http/documentation#files-upload>
 
 -}
-upload : UserAuth -> UploadRequest -> Task UploadError UploadResponse
+upload : UserAuth -> UploadRequest -> Task UploadError FileMetadata
```

Comment:
Yeah, I think this is good to get rid of `UploadResponse`.  I didn't notice before that the response had a specific type name (FileMetadata) in the API docs.